src/chia_log/handlers/daily_stats/stats_manager.py

```diff
@@ -82,11 +90,16 @@ def consume_signage_point_messages(self, objects: List[FinishedSignagePointMessa
                 for obj in objects:
                     stat_acc.consume(obj)
 
-    def _send_daily_notification(self):
-        summary = f"Hello farmer! 👋 Here's what happened in the last {self._frequency_hours} hours:\n"
+    def _send_daily_notification(self, startup: bool = False):
```

Comment:
Another approach would be to copy this code to split this code into `_send_daily_notification()` and `_send_test_notification()`. Not sure yet which I prefer.

Reply:
I would probably have left it without if-else statements for simplicity. Not resetting the stats only adds another 30 seconds of data for the next daily notification which isn't substantial to sweat over and the summary text change maybe nice but it'll be obvious either way I think.

Anyway if you decide to keep it with the extra handling, I suggest to directly read the `self._startup` variable without passing as parameter and reset it here as well, so that all handling is in the same function and not also spread over in the `_run_loop`.